File: C_Programming/Practise_Programs/PP_155.c
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_155
//
//  Description       : This program demonstrates the left shift
//                      operation on an unsigned integer.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 18/08/2026
//
//  Time Complexity   : O(1)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

typedef unsigned int UNIT;

int main()
{
    UNIT iMask = 0x1;
    int iCnt = 0;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    printf("%d\n", iMask);  
    iMask = iMask << 1;

    return 0;
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      No input required.
//
//  Sample Output :
//
//      1
//      2
//      4
//      8
//      16
//      32
//      64
//      128
//      256
//      512
//      1024
//
//////////////////////////////////////////////////////////////////

File: Java_Programming/Practise_Programs/PP_155.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_155
//
//  Description       : This program accepts a number from the
//                      user and displays a pattern of stars and
//                      hash symbols based on the given number.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 03/09/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

import java.util.*;

class PP_155
{
    public static void Display(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print("*\t#\t");
        }
        System.out.println();
    }
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the elements : ");
        iValue = sobj.nextInt();

        Display(iValue);
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Enter the elements : 5
//
//  Sample Output :
//
//      *	#	*	#	*	#	*	#	*	#
//
//////////////////////////////////////////////////////////////////